geomesa-tools/bin/install-hadoop-accumulo.sh

```diff
@@ -49,6 +52,7 @@ else
             "${base_url}commons-logging/commons-logging/${com_log_version}/commons-logging-${com_log_version}.jar"
             "${base_url}com/google/guava/guava/${guava_version}/guava-${guava_version}.jar"
             "${base_url}org/apache/commons/commons-vfs2/${commons_vfs2_version}/commons-vfs2-${commons_vfs2_version}.jar"
+            "${base_url}org/apache/htrace/htrace-core/${htrace_version}/htrace-core-${htrace_version}.jar"
```

Comment:
@acerioni can you add an argument  for 1.6 vs 1.7 in the script perhaps and then select the list of users from that?